docs/tasks/administer-cluster/change-default-storage-class.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Changing the default StorageClass
+---
+
+{% capture overview %}
+This page shows how to change the default Storage Class that is used to
+provision volumes for PersistentVolumeClaims that have no special requirements.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Why change default storage class
+
+Depending on the installation method, Kubernetes cluster may be deployed with
+existing StorageClass that is marked as default. This default StorageClass
+is then used to dynamically provision storage for PersistentVolumeClaims
+that do not require any specific storage class. See
+[PersistentVolumeClaim documentation](/docs/user-guide/persistent-volumes/#class-1)
+for details.
+
+This pre-installed default StorageClass may not fit well with your expected work
+load, e.g. it can provision too expensive storage, and you may choose to either
+change the StorageClass or disable it completelly to avoid dynamic provisioning
+of storage.
+
+Simply deleting the default StorageClass may not work, as it may be re-created
+automatically by addon manager running in your cluster. Consult installation
+guide for details about addon manager and how to disable individual addons.
+
+## Changing default StorageClass
+
+1. List the StorageClasses in your cluster:
+
+        kubectl get storageclass
+
+    The output is similar to this:
+
+        NAME                TYPE
+        default (default)   kubernetes.io/gce-pd
+        gold                kubernetes.io/gce-pd
+
+    Default StorageClass is marked by `(default)`.
+
+1. Mark the default StorageClass as non-default:
+
+   Default StorageClass has annotation
+   `storageclass.beta.kubernetes.io/is-default-class` set to `true`. Any other value
```

Comment:
aside: we should move this to a non-beta annotation with an appropriate name for next release